OutOfSchool/OutOfSchool.WebApi/Services/ProviderService.cs

```diff
@@ -365,7 +368,15 @@ public async Task<ResponseDto> Block(ProviderBlockDto providerBlockDto)
             false,
             false);
 
+        logger.LogInformation("Block/Unblock the particular provider admins and deputy providers belonging to the Provider starts.");
+
+        _ = await providerAdminService
```

Comment:
We've discussed this code at our meeting. But we didn't have a common point of view on what we had to do in case some of the provider admins or deputies wouldn't be blocked. We decided to do additional checks on other controllers. 
Create TODO.

Reply:
Thanks for the explanation.